qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -51,13 +52,13 @@
         - check_sync_delimited:
             gagent_check_type = sync_delimited
         - check_set_user_password:
-            no Windows
             gagent_check_type = set_user_password
             image_snapshot = yes
             variants:
                 - no_crypto:
-                    new_password = "abcdef"
+                    new_password = "Abc_123"
```

Comment:
any  reasons for using this new password ?

Reply:
@hereischen  hi,
why can't leave '.' in commit title?
The reason is in the first comment.
 "2.Change new password to a complex one for windows requirment. "

Thanks.